vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc

```diff
@@ -57,8 +57,9 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - systemctl enable --now docker kubelet
       - if (curl -sk --max-time 10 https://{{ CLUSTER_APIENDPOINT_HOST }}:6443/healthz); then echo \"keepalived already running\";else systemctl start keepalived; fi
       - usermod -aG docker {{ IMAGE_USERNAME }}
-      - systemctl link /lib/systemd/system/monitor.keepalived.service 
-      - systemctl enable --now monitor.keepalived.service
+      - systemctl link /lib/systemd/system/monitor.keepalived.service
+      - systemctl enable monitor.keepalived.service
+      - systemctl start monitor.keepalived.service
```

Comment:
Shouldn't it work with --now ?

Reply:
it should, same outcome as enable + start

Reply:
Then please do not change it

Reply:
I am still struggling to understand the need to do this change

Reply:
the outcome is the same. I would like to merge this and it can be changed later.